package/yast2-country.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-country
-Version:        3.1.30
+Version:        3.1.30.1
```

Comment:
The version is wrong. For SLE 12 SP2, you should use `3.1.31` (3-digits number) as `master` is currently `3.2.1` (it won't collide).